File: src/targets/python/requests/fixtures/application-form-encoded.py
import requests

url = "https://httpbin.org/anything"

payload = {
    "foo": "bar",
    "hello": "world"
}
headers = {"content-type": "application/x-www-form-urlencoded"}

response = requests.post(url, data=payload, headers=headers)

print(response.text)

File: src/targets/javascript/jquery/fixtures/application-form-encoded.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: {
    foo: 'bar',
    hello: 'world'
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});